IdentityCore/tests/integration/MSIDTokenCacheDataSourceIntegrationTests.m

```diff
@@ -185,7 +185,8 @@ - (void)testRemoveItemWithKey_whenKeyIsValidWithType_shouldRemoveItem
 {
     MSIDTokenCacheItem *token = [MSIDTokenCacheItem new];
     token.accessToken = @"some token";
-    MSIDTokenCacheKey *key = [[MSIDTokenCacheKey alloc] initWithAccount:@"test_account" service:@"test_service" generic:self.generic type:nil];
+    token.tokenType = MSIDTokenTypeAccessToken;
```

Comment:
This unit test function was exactly the same as `testRemoveItemWithKey_whenKeyIsValid_shouldRemoveItem`, so I assume it was copy-pasted but forgotten to modify :)